app/controllers/templates_controller.rb

```diff
@@ -22,8 +22,26 @@ def new_child
   def delete_child
   end
 
+  def speakers_search
+    @speakers = Speaker.canonical
+    @speakers = @speakers.ft_search(search_query) if search_query
+  end
```

Comment:
small suggestion here, when we click on the field there is a first query `Speaker.canonical` returns about 3k+ elements and renders them as options that are turbo streamed to the view. 

while this is not a high traffic page the perf don't really matter but as our number of speaker grow over time this will also increase and create bloat queries.

I think we could limit to a certain amount speakers (200?)  WDYT?

Reply:
Great suggestion and good catch! Will fix 👍